code/android-media-library/src/main/java/com/adobe/marketing/mobile/media/MediaCollectionTracker.java

```diff
@@ -1233,6 +1241,9 @@ class ErrorMessage {
             "Media tracker is not tracking the State passed into 'API:trackEvent(StateEnd)'.";
     static final String ErrTrackedStatesLimitReached =
             "Media tracker is already tracking maximum allowed states (10) per session.";
+    static final String ErrInvalidErrorId =
+            "ErrorId passed into 'API:trackError' is invalid. Please pass valid non-empty non-null"
+                    + " string for ErrorId.";
```

Comment:
Can we merge the error msg in one string?

Reply:
The code formatter formats it this way and should be fine.